software/moltres/wiki/input_example.md

```diff
@@ -50,29 +63,64 @@ application for visualizing output files is
 [VisIt](https://wci.llnl.gov/simulation/computer-codes/visit/) or
 [yt](http://yt-project.org/) may also be used.
 
-Ok, running through the input file from the top:
+#### Model Geometry
+
+The domain for the 2-D MSRE model is a 72.5 cm by 151.75 cm rectangle that is
+slightly smaller than the total height of the core, and includes 1/2 the width
+of the core, extending from the core center line on the left to the core wall on
+the right. For this steady-state simulation, the core center line is a symmetry
+boundary. The domain models 14 fuel channels and alternates between regions that
```

Comment:
It looks like this edit is not complete. The full sentence starting with "The domain models ..." needs to be deleted, along with the nest sentence starting with "In the figure..." since this is incorporated in the modified text.

Reply:
Actually, I see below that these were deleted one at a time. This makes the commit more confusing, but maybe it was the easiest way to make the change?